extensions/interactions/LogicProof/static/js/studentSpec.js

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Unit tests for LogicProof interaction student components.
  */
 
-var errorWrapper = function(
+errorWrapper = function(
```

Comment:
Should this and other stuff in this file not be in global scope?

Reply:
Fixed.